src/App.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import "./App.css";
+import { GiphyFetch } from "@giphy/js-fetch-api";
+import { SearchResults } from "./SearchResults";
+import { Paginator } from "./Paginator";
+
+const gf = new GiphyFetch(process.env.REACT_APP_GIPHY_API_KEY);
+
+function App() {
+  const [searchTerm, setSearchTerm] = React.useState();
+  const [activeSearch, setActiveSearchTerm] = React.useState();
+  const [currentPage, setCurrentPage] = React.useState(0);
+
+  const [results, setResults] = React.useState();
+  React.useEffect(() => {
+    if (activeSearch !== undefined) {
+      gf.search(activeSearch, { offset: currentPage * 5, limit: 5 }).then(
```

Comment:
This confused a lot of people when I was walking through this live, which surprised me.  But just to reiterate what I said before, there's no tangible difference between this approach and the following:

```suggestion
      axios.get(
        'http://api.giphy.com/v1/gifs/search',
        {
          params: {
            q: activeSearch,
            api_key: process.env.REACT_APP_GIPHY_API_KEY,
            offset: currentPage * 5,
            limit: 5,
          }
        }
      ).then(
```

If you feel more comfortable with axios and fetch, there's nothing wrong with those approaches.  I just used this sdk because "it was there" and it seemed easier than manually building the HTTP myself.